packages/suite/.stylelintignore

```diff
@@ -2,5 +2,6 @@
 # files below doesn't contains any styles so it's safe to ignore them
 
 src/components/suite/index.tsx
+src/components/suite/Modal/index.tsx
```

Comment:
just a nitpick, this will ignore whole file, so if we later add some styling to it it won't get linted and nobody will remember this is the reason. I would prefer to disable stylelint just for single line that are causing a problem.

Reply:
For some reason it doesn't stop failing with inline disable rule. It still fails with `Unexpected token`, so I keep it in packages/suite/.stylelintignore